### PR TITLE
Add support for customizing file name via config file. #107 #110 #111

### PR DESCRIPTION
**Pull request by jmathai**
…d get tests to pass. #107 #110

### CODE REVIEW

File: elodie/tests/filesystem_test.py [hunk 1]
```diff
@@ -205,43 +205,43 @@ def test_get_file_name_with_original_name_title_exif():
 
     assert file_name == helper.path_tz_fix('2015-12-05_00-59-26-foobar-foobar-title.jpg'), file_name
 
-def test_get_folder_path_plain():
+def test_get_path():
```
Comment (adamantike) [inline elodie/tests/filesystem_test.py]: These three `test_get_path` tests should have different names!

Comment (jmathai) [inline elodie/tests/filesystem_test.py]: That's definitely a vim key combination screw up. Good catch.

Comment (jmathai) [inline elodie/tests/filesystem_test.py]: Fixed.

**Comment by jmathai** [comment]
Closing this in favor of #225.

**Pull request closed without merging**